modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -1951,7 +1951,13 @@ private void updateDisabled() {
      * specified selector, this function returns the first of them.
      * <p>
      *     For example, if a Node is given the id of "myId", then the lookup method can
-     *     be used to find this node as follows: <code>scene.lookup("#myId");</code>.
+     *     be used to find this node as follows: {@code scene.lookup("#myId");}.
+     * </p>
+     * <p>
+     *     If two nodes, lets say NodeA and NodeB have same style class "myStyle" and NodeA has
```

Comment:
Since this is a separate paragraph, please add "For example, " here to make it clear that this is an example (and not normative specification). Maybe something like this:

```
     * For example, if two nodes, NodeA and NodeB, have the same ...
```